src/js/wrap.cpp

```diff
@@ -90,17 +115,32 @@ namespace ccf::js
     }
     size_t stack_size = 1024 * 1024;
     size_t heap_size = 100 * 1024 * 1024;
+    std::chrono::seconds default_max_execution_time{240};
+
     const auto jsengine = tx->ro<ccf::JSEngine>(ccf::Tables::JSENGINE);
     const std::optional<JSRuntimeOptions> js_runtime_options = jsengine->get();
 
     if (js_runtime_options.has_value())
     {
       heap_size = js_runtime_options.value().max_heap_bytes;
       stack_size = js_runtime_options.value().max_stack_bytes;
+      default_max_execution_time =
+        std::chrono::seconds{js_runtime_options.value().max_execution_time};
     }
 
     JS_SetMaxStackSize(rt, stack_size);
     JS_SetMemoryLimit(rt, heap_size);
+    UntrustedHostTime* host_time = new UntrustedHostTime();
```

Comment:
I don't think this pointer is freed anywhere here?

Reply:
It was just being set to `NULL`, it is now deleted too